docs/how-to-guides/enqueueing-assets-in-the-editor.md

```diff
@@ -29,8 +29,8 @@ function example_enqueue_editor_assets() {
         'example-editor-scripts', 
         plugins_url( 'editor-scripts.js', __FILE__ ) );
     }
-    wp_enqueue_style( 
-        'example-editor-styles', 
+    wp_enqueue_style(
+        'example-editor-styles',
         plugins_url( 'editor-styles.css', __FILE__ ) );
     }
```

Comment:
Why is this closing `}` here?

Reply:
Yeah, those were blatant typos 🤦‍♂️ All fixed.